src/web/app/components/question-submission-form/question-submission-form.component.html

```diff
@@ -173,10 +173,19 @@
       ></tm-constsum-recipients-question-constraint>
     </div>
 
-    <div class="alert alert-warning" role="alert" *ngIf="model.recipientList.length === 0">
+    <div class="alert alert-warning" role="alert" *ngIf="!model.recipientList.length && model.isLoaded">
       <span *ngIf="model.recipientType === FeedbackParticipantType.OWN_TEAM_MEMBERS">This question is for team members and you don't have any team members. Therefore, you will not be able to answer this question.</span>
       <span *ngIf="model.recipientType === FeedbackParticipantType.TEAMS">This question is for other teams in this course and this course doesn't have any other team. Therefore, you will not be able to answer this question.</span>
       <span *ngIf="model.recipientType === FeedbackParticipantType.STUDENTS">This question is for other students in this course and this course doesn't have any other student. Therefore, you will not be able to answer this question.</span>
     </div>
+
+    <div class="row" *ngIf="model.recipientList.length > 0">
+      <div class="col-12 text-center">
+        <button id="btn-submit-qn-{{ model.questionNumber }}" type="submit" class="btn btn-success"
+                ngbTooltip="You can save your responses at any time and come back later to continue."
+                (click)="saveFeedbackResponses()" [disabled]="isSavingResponses || isSubmissionDisabled"
+        ><tm-ajax-loading *ngIf="isSavingResponses"></tm-ajax-loading>Submit Responses</button>
```

Comment:
Should we update the tooltip for question-level submit buttons to something like `Save your responses for this question`, in case the user made some changes to other questions and closes the tab after submitting responses for only one question, thinking that it will save responses for all questions. However I think this possibility is much lower given that we have the modal.

Otherwise, LGTM.

Reply:
Good suggestion. Updated the message.